src/tests/gssapi/t_ccselect.py

```diff
@@ -31,12 +31,16 @@
 
 host1 = 'p:' + r1.host_princ
 host2 = 'p:' + r2.host_princ
+foo1 = 'foo.krbtest.com'
+foo2 = 'foo.krbtest2.com'
 
-# gsserver specifies the target as a GSS name.  The resulting
+# gssserver specifies the target as a GSS name.  The resulting
 # principal will have the host-based type, but the realm won't be
 # known before the client cache is selected (since k5test realms have
 # no domain-realm mapping by default).
-gssserver = 'h:host@' + hostname
+gssserver = 'h:host@' + foo1
```

Comment:
Perhaps "foo1" should be "foo" if we're going to use "gssserver" and "gssserver2".

Also, the comment above isn't really accurate after the fallback realm changes, and should be clarified.